cellfinder/core/classify/resnet.py

```diff
@@ -14,7 +13,8 @@
     MaxPooling3D,
     ZeroPadding3D,
 )
-from tensorflow.keras.optimizers import Adam, Optimizer
+from keras.optimizers import Adam, Optimizer
+from tensorflow import Tensor
```

Comment:
This looks to be the last remnant of directly importing from tensorflow, naively can this be a KerasTensor now? 

```suggestion
from keras.engine.keras_tensor import KerasTensor
```

Reply:
yes! I only realised this now when doing platform agnostic tests with jax, but happy to implement in this PR

Reply:
Perfect, I'll merge this now!

Reply:
ok, will be part of PR #378 :)